sdk/src/main/scala/com/horizen/SidechainApp.scala

```diff
@@ -434,15 +423,28 @@ class SidechainApp @Inject()
 
     Http().bindAndHandle(combinedRoute, bindAddress.getAddress.getHostAddress, bindAddress.getPort)
 
-    //on unexpected shutdown
+    //Retrieve the list of all registered Shutdown hooks
+    val clazz = Class.forName("java.lang.ApplicationShutdownHooks")
+    val field = clazz.getDeclaredField("hooks")
+    field.setAccessible(true)
+    val hooks = field.get(null)
+
+    //Remove the Logger shutdown hook
+    val threadMap = hooks.asInstanceOf[JMap[Thread, Thread]].values()
+    threadMap.asScala.toArray.find(thread => thread.getName.equals("Logging-Cleaner")) match {
+      case Some(thread) =>
+        Runtime.getRuntime.removeShutdownHook(thread)
+      case None =>
+    }
```

Comment:
Is it not enough just to set `shutdownHook="disable"` in the logger xml file?
I am a bit worried that we point to the name `Logging-Cleaner`, that can be changed inside the library.
What do you think about this comment https://stackoverflow.com/a/48063926 ?

Reply:
It's not enought to use the config file.
With the approach described here  https://stackoverflow.com/a/48063926 the Logging-Cleaner Thread is not removed from the list of the ShutdowHook thread.
Beside that i agree with you that pointing to a specific Thread name is not a good practice.

I would like to add that also removing the `shutdownHook="disable"` breaks the correct behaviour.

Reply:
within the current approach on the node startup I can see the batch of new warnings:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.horizen.SidechainApp (file:/home/oleksandr/Dev/Sidechains-SDK/sdk/target/classes/) to field java.lang.ApplicationShutdownHooks.hooks
WARNING: Please consider reporting this to the maintainers of com.horizen.SidechainApp
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```
it doesn't seems good to me.

Reply:
Looks good with the new solution